tensorflow/python/ops/gradients_test.py

```diff
@@ -1442,6 +1481,8 @@ def TestFn(input_t):
       out_re = test_fn_re(test_input_t)
       out = TestFn(test_input_t)
 
+    init = tf.compat.v1.global_variables_initializer()
```

Comment:
there's no `tf`; variables.global_variables_initializer() should work

Reply:
Very sorry about that. Fixed it. Thanks